djangocms_version_locking/emails.py

```diff
@@ -0,0 +1,40 @@
+from django.utils.translation import ugettext_lazy as _
+from django.urls import reverse
+
+from cms.utils import get_current_site
+
+from .helpers import send_email
+from .utils import get_absolute_url
+
+
+def notify_version_author_version_unlocked(version, unlocking_user):
+    # If the unlocking user is the current author, don't send a notification email
+    if version.created_by == unlocking_user:
+        return
+
+    site = get_current_site()
+    recipients = [version.created_by]
+    subject = "[Django CMS] ({site_name}) {page_title} - {description}".format(
+        site_name=site.name,
+        page_title=version.content,
```

Comment:
Doesn't have to be a Page, so `page_title` is misleading

Reply:
Agreed.